docs/how-to/deploy/deploy-spaces.md

```diff
@@ -0,0 +1,53 @@
+# Deploy on Juju spaces
```

Comment:
Question: Are you sure you want this to be a deploy guide? It doesn't seem to be a new, distinctive way of deploying Kafka.
I think it might fit better as a How-to use Juju space guide or even just an explanation page, explaining the specific usage of this feature for Kafka.

Reply:
OK, let's leave it as it is right now. But I'd like to discuss this approach later